Dockerfile

```diff
@@ -62,6 +66,12 @@ RUN wget https://bootstrap.pypa.io/get-pip.py -O - | python3
 RUN wget https://bootstrap.pypa.io/get-pip.py -O - | python
 RUN pip install virtualenv httpie
 
+# Install rustup
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | \
+        CARGO_HOME=/opt/cargo/ sh -s -- -y && . /opt/cargo/env
+
+ENV PATH="${PATH}:/opt/cargo/bin"
+
```

Comment:
I would not include this tool in Docker image itself. Please move installation either into a `spksrc.cross-rust.mk` Makefile, or in `native/rustup`, the same way Go compiler is downloaded and compiled.